pkgs/servers/jellyseerr/default.nix

```
@@ -85,12 +78,12 @@ mkYarnPackage rec {
     homepage = "https://github.com/Fallenbagel/jellyseerr";
     longDescription = ''
       Jellyseerr is a free and open source software application for managing
-      requests for your media library. It is a a fork of Overseerr built to
+      requests for your media library. It is a fork of Overseerr built to
       bring support for Jellyfin & Emby media servers!
     '';
     license = licenses.mit;
     maintainers = with maintainers; [ camillemndn ];
     platforms = platforms.linux;
```

```suggestion
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ camillemndn ];
    platforms = lib.platforms.linux;
```
(removing `with lib;`)